### ISSUE TITLE
Friendlier Error Messsages 

### ISSUE THREAD

**Issue by barnjamin** [algorand/pyteal#127]
## Problem

Couple reports in the discord with confusion around including a call like `App.localGetEx` return var in the seq.  The error message is something like:

```
raise TealInternalError(msg) from errors[0]
pyteal.TealInternalError: Encountered 1 error when assigning slots to subroutine
```

This error message makes it difficult to figure out the actual cause

## Solution

Not sure how it could be improved 

## Urgency

No urgency

**Comment by algoanne** [comment]
@barnjamin is this still an issue?

**Comment by barnjamin** [comment]
Yes but maybe less so after improving the traceback on pyteal generated expressions as in MultiValues
